### DISCUSSION TITLE
Documentation Update

### DISCUSSION THREAD

**Discussion by ghost** [fastapi/fastapi#7340] [Questions] [answered]
### Some suggestions for the documentation

* [ ]  Use a more open layout like gitbook docs does
* [ ]  Show a tutuorial to make a prject like a simple blog api so that they can reference it for a bigeer project
* [ ]  Start a discord and slack channel
* [ ]  Post code on gitlab apart from github as well so that people from gitlab can help you too..

**Comment by Kludex** [comment]
>  Start a discord and slack channel

We have a Gitter integration: https://gitter.im/tiangolo/fastapi. But personally I'd like to have a discord instead/also. @tiangolo what do you think? Community is growing fast and discord has a better setup to have discussions, besides gitter have some problems on mobile and Ubuntu versions.

>  Show a tutuorial to make a prject like a simple blog api so that they can reference it for a bigeer project

We already have it on the documentation.

>  Use a more open layout like gitbook docs does

I have no idea what you're talking about.

>  Post code on gitlab apart from github as well so that people from gitlab can help you too..

I have no idea what you're talking about.

**Comment by ghost** [comment]
Hi, by open layout i mean something like this:
[https://docs.masoniteproject.com/](url) This is much more pleasing and beautiful to the eye.

Tutorial about making a small project. Where's the url? Also can some official screen casts be made so that we can use them as reference. People understand better from videos.

By posting code on gitlab as well as github, I mean that committhe same framework code to [gitlab.com](url) as well.
This will increase the community

> And having a community on slack and discord is absolutely essential for the framework community to grow.

**Comment by alecgerona** [comment]
Docs aside, here's a +1 for discord integration. It's _very_ helpful for getting (near) real-time support and getting up-to-date on notifications.

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest and the discussion here, everyone.

About the docs, the preference is quite personal. I like how they are now.

About the tutorial, the docs are made in the style of a tutorial. If the original idea was to make a full course, yeah, that would be great, the same for the videos... I would just have to find the time and resources. :shrug: 

Discord is here thanks to @Kludex: https://fastapi.tiangolo.com/help-fastapi/#join-the-chat :rocket: 

About GitLab, no thanks. That would mean even more work for me. :sweat: 

...now, given the original poster's account was deleted (it's a @ghost now), I'm gonna close this issue.
